docs/animations/Fade.md

```diff
@@ -42,6 +42,28 @@ Behavior animations can also be chained and awaited.
 
 [Fade Behavior Sample Page Source](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Fade)
 
+## InterpolationType
```

Comment:
Just a question, why did you decide to go with InterplationType vs EasingType for the naming? EasingType seems more approachable, but that might be just me.